workflows_core/workflow/context_manager.py

```diff
@@ -99,4 +99,5 @@ def _set_status(self, status: str):
             workflow_name=self._workflow_name,
             additional_information=self._additional_information,
             send_email=self._send_email,
+            worker_number=worker_number
```

Comment:
but we can here?

Reply:
This isn't what's being fed through to the API.